package.json

```diff
@@ -31,20 +31,19 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "acorn": "^6.1.1",
+    "acorn": "^6.4.2",
     "clean-webpack-plugin": "^2.0.2",
     "path": "^0.12.7",
-    "ts-loader": "^6.0.1",
-    "typescript": "^3.4.5",
-    "webpack": "^4.31.0",
-    "webpack-bundle-analyzer": "^3.3.2",
-    "webpack-cli": "^3.3.2"
+    "ts-loader": "^6.2.2",
+    "typescript": "^3.9.7",
+    "webpack": "^4.46.0",
+    "webpack-bundle-analyzer": "^3.9.0",
+    "webpack-cli": "^3.3.12"
   },
   "dependencies": {
-    "@types/react": "^16.9.11",
-    "@types/redux": "^3.6.0",
-    "react": "^16.8.6",
-    "redux": "^4.0.1"
+    "@types/react": "^17.0.0",
+    "react": "^17.0.1",
```

Comment:
Bumping react to 17 in deps (and not in peerDeps) could cause discrepancies and also cause breaking changes.